### PR TITLE
Missed LIBPROTOC_EXPORT for GRPC added

### PR DESCRIPTION
**Pull request by KindDragon**

### CODE REVIEW

**Comment by KindDragon** [comment]
Fix #2219

File: src/google/protobuf/compiler/objectivec/objectivec_helpers.h [hunk 1]
```diff
@@ -87,8 +87,8 @@ string FileClassName(const FileDescriptor* file);
 
 // These return the fully-qualified class name corresponding to the given
 // descriptor.
-string ClassName(const Descriptor* descriptor);
-string ClassName(const Descriptor* descriptor, string* out_suffix_added);
+string LIBPROTOC_EXPORT ClassName(const Descriptor* descriptor);
+string LIBPROTOC_EXPORT ClassName(const Descriptor* descriptor, string* out_suffix_added);
 string EnumName(const EnumDescriptor* descriptor);
```
Comment (KindDragon) [inline src/google/protobuf/compiler/objectivec/objectivec_helpers.h]: Ok, what with other files?

Comment (xfxyjwf) [inline src/google/protobuf/compiler/objectivec/objectivec_helpers.h]: Yeah, please add LIBPROTOC_EXPORT to all public functions in charp_names.h as well. Right now the GetOutputFile() function in csharp_names.h is also missing it.

**Comment by KindDragon** [comment]
@xfxyjwf any news with this PR?

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by xfxyjwf** [comment]
Thanks, the change looks good.

**Pull request merged**